packages/web3-eth-contract/src/types.ts

```diff
@@ -187,7 +187,7 @@ export interface NonPayableMethodObject<Inputs = unknown[], Outputs = unknown[]>
 	 */
 
 	call<SpecialOutput = Outputs>(
-		tx?: NonPayableCallOptions,
+		tx?: Omit<NonPayableCallOptions, 'nonce'>,
```

Comment:
It looks like `nonce` is allowed for `call`
<img width="412" alt="image" src="https://user-images.githubusercontent.com/20375610/233437711-dcc005ff-cb30-4ac2-8bba-bdcaa1b43381.png">

Reply:
These screenshots are coming from the [JSON RPC spec](https://ethereum.github.io/execution-apis/api-documentation/)